core/src/main/java/org/infinispan/xsite/OfflineStatus.java

```diff
@@ -158,6 +120,54 @@ public void amend(Integer afterFailures, Long minTimeToWait) {
       amend(builder.create());
    }
 
+   //this is public for testing purpose only
```

Comment:
A developer using `Ctrl+Q` in IntelliJ will see a regular public method.

I suggest making it package-private and adding an "accessor" class in the same package in the test sources.

Reply:
it would rather make it private and use reflexion ...

Reply:
Sure, that will also work, but it seems more complex to me. Plus it's spelled `reflection` ;)